com.sublimehq.SublimeText.desktop

```diff
@@ -0,0 +1,23 @@
+[Desktop Entry]
+Version=1.0
+Type=Application
+Name=Sublime Text
+GenericName=Text Editor
+Comment=Sophisticated text editor for code, markup and prose
+Exec=/opt/sublime_text/sublime_text %F
+Terminal=false
+MimeType=text/plain;
+Icon=sublime-text
```

Comment:
this is the wrong icon name

Reply:
Sorry, this was the upstream desktop file and I missed a few things. Thanks for the quick feedback!

Reply:
since it's ustream, why is it here?

Reply:
I wanted something pure to start with.

Reply:
Fixed!